ocs_ci/helpers/dr_helpers.py

```diff
@@ -549,3 +561,127 @@ def get_all_drpolicy():
     drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY)
     drpolicy_list = drpolicy_obj.get(all_namespaces=True).get("items")
     return drpolicy_list
+
+
+def enable_fence(drcluster_name):
+    """
+    Once the managed cluster is fenced, all communication
+    from applications to the ODF external storage cluster will fail
+
+    Args:
+        drcluster_name (str): Name of the DRcluster which needs to be fenced
+    """
+
+    logger.info(
+        f"Edit the DRCluster resource for {drcluster_name} cluster on the Hub cluster"
+    )
+    restore_index = config.cur_index
+    config.switch_acm_ctx()
+    fence_params = f'{{"spec":{{"clusterFence":"{constants.ACTION_FENCE}"}}}}'
+    drcluster_obj = ocp.OCP(resource_name=drcluster_name, kind=constants.DRCLUSTER)
+    assert drcluster_obj.patch(
+        params=fence_params, format_type="merge"
+    ), f"Failed to patch {constants.DRCLUSTER}: {drcluster_name}"
```

Comment:
Please raise an exception instead of using assert, the code which uses this function can't decide how to handle the assert failure, which is context dependent.

Reply:
done